exporter/azureblobexporter/metadata.yaml

```diff
@@ -0,0 +1,9 @@
+type: azureblob
+
+status:
+  class: exporter
+  stability:
+    alpha: [traces, metrics, logs]
+  distributions: [observiq]
+  codeowners:
+    active: [atoulme, pdelewski]
```

Comment:
Should probably be `cpheps`

Reply:
I'm going to just delete this as it's not required and we don't put it on any of our other `metadata.yaml` files.